scripts-base/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@mangoweb/scripts-base",
+  "version": "0.0.1",
+  "description": "Simple component model for small to meduim sites. Usable from JS & TS.",
+  "main": "lib/index.js",
```

Comment:
You mean emit `*.mjs` files? Why…?

Reply:
More like a module field with ES6 code, but I'm not sure in what state the eco system is ATM. Reasoning behind it is for reduction of the build step needed, as you import the modules inside you project code anyway.

Reply:
But there's always going to be a build step since the whole thing is written in TypeScript, right? Or do you mean a build step on the implementor's side?

Right now, the actual package contains ES5 (→ no build step necessary for those who want to use it from JS), accompanying TypeScript typings (→ that makes it _practically_ indistinguishable from actual TypeScript), as well as the actual TypeScript source for reference. I'm still not sure what exactly is it that you propose. Could you please be a bit more specific?